### PR TITLE
add /wd4819 to disable C4819 warning, which is treated as error in win32

### PR DESCRIPTION
**Pull request by Kami-996**

### CODE REVIEW

**Comment by ilya-lavrenov** [comment]
@Kami-996 could you please show examples of such warnings in our code?

File: CMakeLists.txt [hunk 1]
```diff
@@ -96,7 +96,7 @@ function(build_ngraph)
     if (UNIX)
         ie_add_compiler_flags(-Wno-error=return-type -Wno-undef)
     elseif(WIN32)
-        set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /wd4308 /wd4146 /wd4703 /wd4244")
+        set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /wd4308 /wd4146 /wd4703 /wd4244 /wd4819")
```
Comment (ilya-lavrenov) [inline CMakeLists.txt:99]: may we need to add this suppression only for 32bits generators?

Comment (Kami-996) [reply CMakeLists.txt:99]: WIN32 is  defined as 1 when the compilation target is x86 or x64 windows programs, not just the 32bits

**Comment by ilya-lavrenov** [comment]
@Kami-996 thanks for the contribution!

**Pull request merged**